frontend/src/config/theme.js

```diff
@@ -58,4 +58,8 @@ export const darkTheme = {
   courseSidebar: {
     menuSubColor: "#262626", // gray-10
   },
+
+  specialsationStep: {
+    background: "#fff",
```

Comment:
Noticed that you're using hex values here but rgb values in other files. Might want to keep everything as the same type for consistency =)